css/style.css

```diff
@@ -0,0 +1,51 @@
+/* html,
+ body {
+   height: 100%;
+   width: 100%;
+   margin: 0;
+   padding: 0;
+ }
+*/
```

Comment:
- [ ] [optional] Kindly do not use comments to remove unwanted code, it is more readable when you delete it. :ok_hand: